news/5525.feature

```diff
@@ -0,0 +1 @@
+The list of RECORD entries is now sorted.
```

Comment:
This should probably be more descriptive, something like:

Ensure that the ``RECORD`` file is sorted when installing from a wheel file.